coalib/coala_main.py

```diff
@@ -100,34 +100,35 @@ def run_coala(console_printer=None,
     """
     all_actions_possible = provide_all_actions()
     apply_single = None
+
     if getattr(args, 'single_action', None) is not None:
         while True:
             for i, action in enumerate(all_actions_possible, 1):
                 console_printer.print(format_lines('{}'.format(
-                    action), symbol='['))
+                    action), symbol='[', line_nr=' ' + str(i) + '. '))
 
             line = format_lines(STR_ENTER_NUMBER, symbol='[')
-
             choice = input(line)
+            choice = str(choice)
 
-            if choice.isalpha():
+            if choice.isalnum():
                 choice = choice.upper()
                 choice = '(' + choice + ')'
-                if choice == '(N)':
+                if choice == '(N)' or choice == '1':
```

Comment:
Do Nothing is for 0 not 1

Reply:
Sorry! I'll change that.

Reply:
if you enter two numbers/options while selecting the option, coala currently performs the same action in that order. can you change your code so that the nature is preserved for now?

Reply:
but since you need to apply single action for now, this seems to be correct for now. Wait, will verify and tell you.

Reply:
umm.. just perform a check for the options user provides, for eg. `A1` should be claimed as an invalid option

Reply:
So i need a print a statement saying that it's an invalid option right?

Reply:
something like line 127-129 of original file